wasm.js

```diff
@@ -0,0 +1,11 @@
+//const fs = require("fs");
+const loader = require("assemblyscript/lib/loader");
+// const compiled = new WebAssembly.Module(fs.readFileSync(__dirname + "/build/optimized.wasm"));
+const promise = fetch("./build/optimized.wasm")
```

Comment:
I don't think this works in nodejs, `fetch` is not defined.
@MaxGraey any suggestions for loading wasm across nodejs + browser?

Reply:
you could use separate code for browser and node. In node just use `fs.readFile` or `fs.readFileSync`. I guess it's most simplest approach. Another one is using something like `isomorphic-fetch` but this add dependency

Reply:
Yeah i think my best option at this point is to run two seperate files for this